### DISCUSSION TITLE
library "librustdesk.so" not found

### DISCUSSION THREAD

**Discussion by devme19** [rustdesk/rustdesk#7314] [Q&A]
Launching lib\main.dart on Android SDK built for x86 in debug mode...
Running Gradle task 'assembleDebug'...
Warning: unexpected element (uri:"", local:"extension-level"). Expected elements are <{}codename>,<{}layoutlib>,<{}api-level>
Warning: unexpected element (uri:"", local:"base-extension"). Expected elements are <{}codename>,<{}layoutlib>,<{}api-level>
√  Built build\app\outputs\flutter-apk\app-debug.apk.
Installing build\app\outputs\flutter-apk\app-debug.apk...
Debug service listening on ws://127.0.0.1:51324/dUBwBP7vD2g=/ws
Syncing files to device Android SDK built for x86...
I/flutter (14231): launch args: []
E/flutter (14231): [ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: Invalid argument(s): Failed to load dynamic library 'librustdesk.so': dlopen failed: library "librustdesk.so" not found
E/flutter (14231): #0      _open (dart:ffi-patch/ffi_dynamic_library_patch.dart:11:43)
E/flutter (14231): #1      new DynamicLibrary.open (dart:ffi-patch/ffi_dynamic_library_patch.dart:22:12)
E/flutter (14231): #2      PlatformFFI.init (package:flutter_hbb/models/native_model.dart:115:26)
E/flutter (14231): #3      initEnv (package:flutter_hbb/main.dart:103:21)
E/flutter (14231): #4      runMobileApp (package:flutter_hbb/main.dart:150:9)
E/flutter (14231): #5      main (package:flutter_hbb/main.dart:43:5)
E/flutter (14231): #6      _runMain.<anonymous closure> (dart:ui/hooks.dart:299:23)
E/flutter (14231): #7      _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:297:19)
E/flutter (14231): #8      _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:184:12)
E/flutter (14231): 
D/ProfileInstaller(14231): Installing profile for com.carriez.flutter_hbb

**Comment by grummbeer** [comment]
Try it with a real Android device. As far as I know, a simulator will not work for dev.
